src/index.js

```diff
@@ -1,11 +1,15 @@
 // @flow
 // An event handler can take an optional event argument
 // and should not return a value
-type EventHandler = (typeOrEvent?: string|any, event?: any) => void;
+type EventHandler = (event?: any) => void;
+type WildCardEventHandler = (type: string, event?: any) => void
+
 // An array of all currently registered event handlers for a type
 type EventHandlerList = Array<EventHandler>;
+type WildCardEventHandlerList = Array<WildCardEventHandler>;
 // A map of event types and their corresponding event handlers.
 type EventHandlerMap = {
+  '*': WildCardEventHandlerList,
```

Comment:
I think this should be:
```js
'*'?: WildCardEventHandlerList
```

Otherwise, it means that `EventHandlerMap` should always have a `"*"` key